### DISCUSSION TITLE
!oMLOGB=1000 not permanent?

### DISCUSSION THREAD

**Discussion by noobtoob4lyfe** [gopro/labs#270] [General]
I've notice that this setting seems to get wiped every time I restart the camera and I need to show it the QR code again.  Is this expected?

**Comment by dnewman-gpsw** [comment]
!oMwxyz... That is not a valid command.  Many docs: https://gopro.github.io/labs/control/tech/

!MLOGB=1000 will work.

**Comment by noobtoob4lyfe** [comment]
I think maybe it has to do with manually entering it into the qrcontrol app.  If you manually enter the text into the app it does not stay permanent for me at least. 
 In a separate issue if you select LOGO from the dropdown, that actually controls the LOGB functionality.  Using that LOGO from the dropdown was how I got LOGB to stay permanent.  I think they might be swapped or named wrong?

**Reply by dnewman-gpsw** [reply]
That is in the old version of the QRControl App. Update it.

And if you used !MLOGB=1000, it is permanent, on any app, as it the camera function, not the apps.
